src/configuration/GlobalConfiguration.cpp

```diff
@@ -92,6 +92,8 @@ const unsigned GlobalConfiguration::GUROBI_NUMBER_OF_THREADS = 1;
 const bool GlobalConfiguration::GUROBI_LOGGING = false;
 #endif // ENABLE_GUROBI
 
+const bool GlobalConfiguration::LP_TIGHTENING_AFTER_SPLIT = true;
```

Comment:
make this a command-line argument and (also add it to the python API)

Reply:
Done.